frame/staking/src/tests.rs

```diff
@@ -346,6 +346,78 @@ fn staking_should_work() {
 		});
 }
 
+#[test]
+fn maximum_validator_set() {
+	ExtBuilder::default()
+		.maximum_validator_count(4)
+		.build()
+		.execute_with(|| {
+			assert_eq!(Staking::maximum_validator_count(), 4);
+		});
+}
```

Comment:
```suggestion
#[test]
fn maximum_validator_set() {
	ExtBuilder::default()
		.maximum_validator_count(4)
		.build()
		.execute_with(|| {
			assert_eq!(Staking::maximum_validator_count(), 4);
		});
}
```

superfluous, this is not testing the logic of the code, this is testing the setup of the tests.

Reply:
We have one test called `basic_setup_works()` or something along those lines in which you can test this.